### PR TITLE
Adding cycle_path_type in PathItem

### PR DESCRIPTION
**Pull request by AurelienLP**
https://jira.kisio.org/browse/TRR-49

For now, we only want to provide information for the type of path used by Bike. BUT we expect to use other types of path for all the other modes (Taxi especially).

### CODE REVIEW

File: response.proto [hunk 1]
```diff
@@ -3,11 +3,20 @@ import "type.proto";
 
 package pbnavitia;
 
+enum PathType {
```
Comment (pbougue) [inline response.proto]: Is it just for cycles? In this case CyclePathType could be a better name.

File: response.proto [hunk 2]
```diff
@@ -3,11 +3,20 @@ import "type.proto";
...
+    Road = 0;
+    SharedCycleWay = 1;    // Shared use lane (could be shared with pedestrians)
+    DedicatedCycleWay = 2; // Dedicated cycle lane
+    SeparatedCycleWay = 3; // A separate cycle lane (physical separation from the main carriageway
+    Residential = 4;
+}
```
Comment (pbougue) [inline response.proto]: Will there be an "OppositeCycleWay" to model rue de Charenton for example?

File: response.proto (hunk 1 above)
Comment (AurelienLP) [reply response.proto]: For now we want to use it only for cycles. BUT I am sure we will use other types for the other modes.

File: response.proto (hunk 2 above)
Comment (AurelienLP) [reply response.proto]: Valhalla doesn't have this information yet. Maybe someday...

**Pull request merged**